website/content/docs/install-boundary/deploy.mdx

```diff
@@ -1,18 +1,27 @@
 ---
 layout: docs
-page_title: Install Boundary
+page_title: Deploy Boundary
 description: >-
- Install a self-managed version of Boundary Enterprise or Community Edition on Ubuntu/Debian, CentOS/RHEL, or Amazon Linux using the binary file.
+ Deploy a self-managed version of Boundary Enterprise or Community Edition on Ubuntu/Debian, CentOS/RHEL, or Amazon Linux by installing the binary file.
 ---
 
-# Install Boundary
+# Deploy Boundary
+
+To deploy a self-managed Boundary environment you should:
+
+1. Deploy and configure Boundary controllers and workers
+2. Install end user clients
 
 This guide outlines the required steps to manually install and configure a
 single HashiCorp Boundary cluster as defined in the [Recommended
 architecture](/boundary/docs/install-boundary/recommended-architecture) topic.
 It assumes you install Boundary on virtual machines (VMs) or bare-metal servers
 running a Debian or Red Hat-based Linux distribution.
 
+You can also deploy Boundary using Terraform. Refer to the [Terraform patterns](/boundary/docs/install-boundary/terraform-patterns) page to learn more.
```

Comment:
If I go to the Terraform patterns page, it looks like it showing me how to configure Boundary after its been deployed, right? It's not showing me how to deploy Boundary with TF.

Reply:
yeah you're totally right, I'll get that fixed 👍